pkg/volume/projected/projected_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"testing"
 
 	authenticationv1 "k8s.io/api/authentication/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Can you add some tests here in:

https://github.com/kubernetes/kubernetes/blob/429aefa2e6e1d41cab892703733cf6fdd9919561/pkg/volume/projected/projected_test.go#L717

Specifically please cover FsUser != nil, FsGroup != nil, FsUser != nil && FsGroup != nil, and DefaultMode == nil.

Reply:
done